components/public/ConfigurationModal.js

```diff
@@ -5,60 +5,45 @@ const ConfigurationModal = ({ modalProp, toggle }) => {
   return (
     <div>
       <Modal isOpen={modalProp} toggle={toggle}>
-        <ModalHeader
-          toggle={toggle}
-          className="d-flex flex-columns border-none bg-brand-primary text-white"
-        >
+        <ModalHeader toggle={toggle} className="d-flex flex-columns border-none bg-brand-primary text-white">
           <span className="font-weight-bold h3">Welcome to Mart</span>
         </ModalHeader>
         <ModalBody className="py-4">
-          <p>👋 Hey there,</p>
           <p>
-            Thank you for downloading MART and getting started with NCR's APIs.
-            In order for this application to run, there are a couple tasks that
-            need to be completed.
+            <span role="img" aria-label="Wave">
+              👋
+            </span>{' '}
+            Hey there,
           </p>
```

Comment:
If the idea with the prettify formatting was to have in-line styling then I think it would make more sense to do: 
`            <span role="img" aria-label="Wave">👋 </span> {' '}
             Hey There,
`

This isn't an important detail but it just seemed weird to me.

Reply:
It looks like prettier wants it to be on 3 lines for some reason. I did remove the extra `{' '}` but I leave that other formatting up to prettier (format on save is so nice)